static/src/javascripts/projects/common/modules/social/share-count.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+import fastdom from 'fastdom';
+import reportError from 'lib/report-error';
+import $ from 'lib/$';
+import fetchJSON from 'lib/fetch-json';
+import config from 'lib/config';
+import { integerCommas } from 'lib/formatters';
+import template from 'lodash/utilities/template';
+import { inlineSvg } from 'common/views/svgs';
+import shareCountTemplate
+    from 'raw-loader!common/views/content/share-count.html';
+import shareCountImmersiveTemplate
+    from 'raw-loader!common/views/content/share-count-immersive.html';
+
+const $shareCountEls = $('.js-sharecount');
+const tooltip: string = 'Facebook: <%=facebook%>';
+const counts: Object = {
+    facebook: 'n/a',
+};
+let shareCount: number = 0;
+let $fullValueEls;
+let $shortValueEls;
+
+const incrementShareCount = (amount: number): void => {
+    if (amount !== 0) {
+        shareCount += amount;
+
+        const displayCount = parseInt(shareCount.toFixed(0), 10);
```

Comment:
Not sure about whether we really need it - `1.5.toFixed(0) === 2` whereas `parseInt(1.5, 10) === 1`. I didn't want to change that and so kept the existing notation.